client/notifications/index.jsx

```diff
@@ -153,6 +154,13 @@ export class Notifications extends Component {
 	render() {
 		const localeSlug = get( user.get(), 'localeSlug', config( 'i18n_default_locale_slug' ) );
 
+		const customMiddleware = {
```

Comment:
in this particular case we probably shouldn't need to _however_ it does depend on `this.props.checkToggle()` which was enough reason for me to show it. this component _shouldn't_ re-render too many times.

another idea I toyed with and felt was overkill at this point is creating the middleware in `componentDidReceiveProps` and update as needed…